.github/workflows/backend-cli-test.yml

```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
       - master
-      - v2.0
+      - v**
```

Comment:
Someone may don't know what happened here, could we add a README instead of multiple comments under .github folder?

Reply:
But they may not read the README, I prefer to write comments directly, although it is more cumbersome.
What do you think?

Reply:
hmmm not sure, `v**` is too mixed to understand, just waiting for others' comments.

Reply:
Why we not develop in `master` or `development`?

Reply:
Good idea, could we merge 2.1 into 2.0 now? We should develop on the master branch indeed. 

cc @mooming

Reply:
en, we should always develop on master branch,

Reply:
yes, 
```
branches:
      - master
```
is good enough.